config/i18n/ru.json

```diff
@@ -103,5 +103,9 @@
     "playlists-big": "В плейлисте слишком много треков ({0} > {1})",
     "playlists-limit": "Треки из плейлиста + треки из вашей очереди достигли максимально доступного лимита ({0} + {1} > {2})",
     "karaoke-enabled": "Режим караоке включен, попробуйте сделать это еще раз, когда он будет выключен!",
-    "left-no-owner-guilds": "Осталось `{}` из-за того что владелец бота не был найден."
-}
\ No newline at end of file
+    "left-no-owner-guilds": "Осталось `{}` из-за того что владелец бота не был найден.",

```

Comment:
Please make sure the indentation of these lines are correct with the rest of the file! Thanks